src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -367,15 +367,22 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
                   onChange={(e) => {
```

Comment:
This isn't something you did wrong, but would you be able to refactor this method so that it reduces the amount of checks? Right now it checks everything, and then if that first if condition isn't satisfied, then it checks each condition individually. This could be refactored to have fewer checks. Something like this:

if file.size >= .... -> throw the error
if file.name.length > 20 -> throw the error
if !regex.test(file.name) -> throw the error
else -> receiptPrepend

lmk if this makes sense. if it doesnt i can hop on a call with you and explain what I'm thinking (it should be a super easy fix). This will make the code more efficient and readable.